pom.xml

```diff
@@ -42,9 +42,13 @@
     </build>
 
     <properties>
+        <main.basedir>${project.basedir}</main.basedir>
```

Comment:
Why don't we use `project.basedir` directly instead of `main.basedir`?

Reply:
you are right, I copied this part from other plugins' repos. I will remove this line 👍